client/src/components/SummaryCalculation.test.ts

```diff
@@ -0,0 +1,36 @@
+import { SAMPLE_ADDRESS_RECORDS } from "state-machine-sample-data";
+import { calculateAggDataFromAddressList, extractLocationDataFromAddr } from "./SummaryCalculation";
+
+describe("extractLocationDataFromAddr()", () => {
+  it("works", () => {
+    expect(extractLocationDataFromAddr(SAMPLE_ADDRESS_RECORDS[0])).toEqual({
+      housenumber: "654",
+      streetname: "PARK PLACE",
+      boro: "BROOKLYN",
+      lat: 40.6737974139504,
+      lng: -73.9562781322538,
+    });
+  });
+});
+
+describe("calculateAggDataFromAddressList()", () => {
+  const agg = calculateAggDataFromAddressList(SAMPLE_ADDRESS_RECORDS);
```

Comment:
Are there any other tests to add here for when certain other values are null and you want to make sure they get summarized appropriately? Like rsloss?

Reply:
Hmmmm I can't think of any right now. Do you have one in mind?